.changeset/bright-zebras-end.md

```diff
@@ -0,0 +1,6 @@
+---
+"astro": minor
+"@astrojs/vercel": patch
+---
+
+add new --viteMode cli flag for dev and build commands
```

Comment:
```suggestion
Adds a new `--viteMode` CLI flag for `dev` and `build` commands to allow you to override the default `development` and `production` modes.
```
Just added some code formatting here, but also I think a statement describing the *usefulness* of this feature can go here! Say what's good about it! (Suggestion above, but correct for accuracy/hype mode. )

You could even add a "This is useful for/when..." second sentence, if you really wanted to.  (For minor releases, we tend to put a little more in the changesets, and then this info is useful to have when we go to write the blog post about the new features.)